docs/source/deployment/airflow_astronomer.md

```diff
@@ -1,22 +1,40 @@
-# How to deploy your Kedro pipeline on Apache Airflow with Astronomer
+# Apache Airflow
 
-This tutorial explains how to deploy a Kedro project on [Apache Airflow](https://airflow.apache.org/) with [Astronomer](https://www.astronomer.io/). Apache Airflow is an extremely popular open-source workflow management platform. Workflows in Airflow are modelled and organised as [DAGs](https://en.wikipedia.org/wiki/Directed_acyclic_graph), making it a suitable engine to orchestrate and execute a pipeline authored with Kedro. [Astronomer](https://docs.astronomer.io/astro/install-cli) is a managed Airflow platform which allows users to spin up and run an Airflow cluster easily in production. Additionally, it also provides a set of tools to help users get started with Airflow locally in the easiest way possible.
+Apache Airflow is a popular open-source workflow management platform. It is a suitable engine to orchestrate and execute a pipeline authored with Kedro because workflows in Airflow are modelled and organised as [DAGs](https://en.wikipedia.org/wiki/Directed_acyclic_graph).
 
-The following discusses how to run the [example Iris classification pipeline](../get_started/new_project.md#create-a-new-project-containing-example-code) on a local Airflow cluster with Astronomer.
+## How to run a Kedro pipeline on Apache Airflow using a Kubernetes cluster
 
-## Strategy
+The `kedro-airflow-k8s` plugin from Get In Data | Part of Xebia enables you to run a Kedro pipeline on Airflow with a Kubernetes cluster. The plugin can be used together with `kedro-docker` to prepare a docker image for pipeline execution.
```

Comment:
We're actually `GetInData | Part of Xebia` not `Get In Data`

Reply:
I think that's what I have, right?

Reply:
`Get In Data | Part of Xebia` != `GetInData | Part of Xebia`

Reply:
Pro tip: maybe we could have a `|GetInData|` substitution that gets automatically expanded to `GetInData | Part of Xebia` 😄 https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#substitutions-with-jinja2

Reply:
We could, but for now, let's just correct in the text